utils/PathSanitizingFileCheck

```diff
@@ -60,9 +60,11 @@ constants.""")
 
     if args.enable_windows_compatibility:
         if args.enable_yaml_compatibility:
-            slashes_re = r'(/|\\\\|\\\\\\\\)'
+            slashes_re = r'(/|\\\\\\\\|\\\\)'
```

Comment:
Why this change?

Reply:
At first I was trying to use the same regular expression for both cases, Having the 2 escaped slashes before the 4 escaped slashes was creating duplicated `/` as a result. Since the extra `/` was not necessary, I finally decided to split it into two regular expressions, and I didn't undo this change.